.github/workflows/build.yml

```diff
@@ -4,53 +4,47 @@ on:
   push:
     branches:
       - main
-  schedule: # Run everyday (after docker-articulate-node-images since we build off those)
+  schedule:
+    # Run everyday (after docker-articulate-node-images since we build off those)
     - cron: "0 1 * * *"
 
 env:
   IMAGE: articulate/articulate-node-images
+  LATEST: 14-stretch-slim
```

Comment:
this is a Q just for me but what is "LATEST"? is that like specific to github workflow or is that another way to write image versions in general?

Reply:
It's an environment variable that's used to determine which image should be tagged as [latest](https://github.com/articulate/docker-articulate-node-images/pull/22/files#diff-5c3fa597431eda03ac3339ae6bf7f05e1a50d6fc7333679ec38e21b337cb6721R43). I put it as a variable instead of in the step so that it would be easy to change which one is the latest